README.md

```diff
@@ -79,6 +79,17 @@ The act() function is a streamlined method for executing single, straightforward
 # Conclusion
 This Artificially Assisted User Interface Testing framework is a pioneering tool in the realm of desktop automation. Its ability to understand and execute a wide range of commands in a natural, intuitive manner makes it an invaluable asset for anyone looking to enhance their productivity and interaction with their Windows environment. It's not just a tool; it's a step towards a future where AI seamlessly integrates into our daily computing tasks, making technology more accessible and user-friendly.
 
+# installation
+```
+cd pywinassistant
+pip install -r .\requirements.txt

```

Comment:
Add step to create virtual environment. This will keep the packages bottled up in a jar.

Reply:
I made an environment with venv using `python -m venv dev` if that helps.